### DISCUSSION TITLE
[Node Operator Question] <FinalizedBlockNumber>

### DISCUSSION THREAD

**Discussion by WM-chainops** [ethereum-optimism/developers#326] [Node Operators]
### Are you running the most up to date node software?

- [X] Yes

### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

When looking up FinalizedBlockNumber in the EN node I built, it continues to be called with the same block number.
Are there any flags that need to be set related to this?

curl rpcurl -k -X POST -H "Content-Type: application/json" -d '{"jsonrpc": "2.0", "method": "eth_getBlockByNumber", "params": ["finalized", false], "id": 0}' 
"number":"0x95a94a"

### Protocol Description

testnet () :  op-geth 1.101308.0-rc.3 / op-node v1.7.1

### Node Logs

/data/op-node/op-node \
        --l1=$L1URL  \
        --l1.rpckind=$L1KIND \
        --l1.beacon=$l1beacon \
	--l1.beacon.ignore=true \
        --l2=ws://localhost:8551 \
        --l2.jwt-secret /data/op-geth/jwt.txt \
        --network=sepolia \
	--network=op-sepolia \
        --rpc.addr 0.0.0.0 \
        --rpc.port 9545 \
	--log.format terminal \
        --log.level info \
	--l1.epoch-poll-interval=1m \

### Additional Information

We restarted the service by setting the --l1.epoch-poll-interval=1m flag, and the FinalizedBlockNumber changed for a while, but it did not change again to the same blocknumber.

The current block number of FinalizedBlockNumber is 9861375, and the block number is from about two days ago.

**Comment by 1195597989** [comment]
Are there any error messages or warnings?

**Reply by WM-chainops** [reply]
no
If you check the log at the time the FinalizedBlockNumber becomes the same, there is no problem.
In addition, I set the l1.epoch-poll-interval=1m flag today, but it had the same problem of stopping, so I changed it to the default value.
